src/USDA.NET/USDA.NET.csproj

```diff
@@ -1,14 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageId>USDA.NET</PackageId>
+    <TargetFramework>netstandard1.1</TargetFramework>
+    <PackageId>USDA.API</PackageId>
     <PackageVersion>0.0.1-beta</PackageVersion>
-    <Authors>ivanmartinvalle</Authors>
+    <Authors>ivanmartinvalle, eonasdan</Authors>
     <Description>.NET client for USDA APIs</Description>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <PackageTags>USDA</PackageTags>
+    <Version>1.7.0.0</Version>
```

Comment:
Where did 1.7.0.0 come from? `PackageVersion` should really be the source of truth.